src/workloads/execution/BackgroundIndexConstruction.yml

```diff
@@ -0,0 +1,156 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload tests the write performance impact of creating indexes on a very large collection.
+  For this workload we perform an index build in parallel with a write workload.
+
+  We measure and care about the index creation time and throughput/latency of write operation
+  (inserts, deletes). Usually we'll see trade-offs in favour of either background index build time
+  or write operations.
+
+  This test synthetically lowers the number of concurrent operations allowed to represent a
+  saturated server.
+
+Keywords:
+- stress
+- indexes
+- InsertRemove
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 1024
+      socketTimeoutMS: 7200000  # = 2 hours
+
+Actors:
+# Phase 1: Insert enough data to ensure than an index build on each field spills to disk with a
+# memory limit of 100MB.
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10_000_000
+    BatchSize: 1000
+    Document:
+      randomInt: {^RandomInt: {min: -10000000, max: 10000000}}
+      randomString: {^RandomString: {length: 16}}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+
+# Phase 2: Lower the memory limit for spilling to disk so that it occurs more often.
+- Name: Setup
```

Comment:
This setup happens in phase 3, while the comment says 2. Does anything happen in phase 2? I see the loader in phase 1, and quiesce and this in phase 3. Was one of those two supposed to be in phase 2?

Reply:
Fixed with the latest commit